modules/components/src/utils/config.js

```diff
@@ -2,7 +2,7 @@ function getValue(key, defaultValue) {
   return (
     process.env[`STORYBOOK_${key}`] ||
     process.env[`REACT_APP_${key}`] ||
-    localStorage[key] ||
+    (typeof window !== 'undefined' && localStorage[key]) ||
```

Comment:
this check is to make sure the packages are usable in an SSR environment, e.g. NextJS